apps/api/src/app/events/services/workflow.queue.service.ts

```diff
@@ -6,6 +6,7 @@ import { QueueNextJob } from '../usecases/queue-next-job/queue-next-job.usecase'
 import { QueueNextJobCommand } from '../usecases/queue-next-job/queue-next-job.command';
 import { JobEntity, JobRepository, JobStatusEnum } from '@novu/dal';
 import { StepTypeEnum, DigestUnitEnum } from '@novu/shared';
+import { FilterQuery } from 'mongoose';
```

Comment:
@ainouzgali I think it's best not to import mongoose in the API folder as we break some of the boundaries we have with @novu/dal. Could you just change it to Partial<JobEntity>? I think it will do the trick instead of this import